optax/_src/utils_test.py

```diff
@@ -61,5 +62,70 @@ def fn(inputs):
         sum(jax.tree_util.tree_leaves(
             jax.tree_util.tree_map(lambda x: x**2, inputs))))
 
+
+class MultiNormalDiagFromLogScaleTest(parameterized.TestCase):
+
+  @staticmethod
+  def _get_loc_scale(loc_shape, scale_shape):
+    loc = 1.5 * jnp.ones(shape=loc_shape, dtype=jnp.float32)
+    scale = 0.5 * jnp.ones(shape=scale_shape, dtype=jnp.float32)
+    return loc, scale
+
+  @parameterized.parameters(
+    (1, 1, 1),
+    (5, 5, 5),
+    ((2, 3), (2, 3), (2, 3)),
+    ((1, 4), (3, 4), (3, 4)),
+    ((1, 2, 1, 3), (2, 1, 4, 3), (2, 2, 4, 3)),
+  )
+  def test_init_successful_broadcast(
+          self, loc_shape, scale_shape, broadcasted_shape):
+    def tuple_shape(shape):
+      if isinstance(shape, tuple):
+        return shape
+      return tuple([shape])
+
+    loc, scale = self._get_loc_scale(loc_shape, scale_shape)
+    dist = utils.multi_normal(loc, scale)
+    self.assertIsInstance(dist, utils.MultiNormalDiagFromLogScale)
+    mean, log_scale = dist.params
+    self.assertTrue(tuple(mean.shape) == tuple_shape(loc_shape))
```

Comment:
Consider using `self.assertEqual(a, b)` instead of `self.assertTrue(a == b)` for better error messages.